strconv/number.mbt

```diff
@@ -206,3 +208,30 @@ fn parse_inf_nan(s : String) -> (Double, Int)? {
   }
   None
 }
+
+fn lsl(a : UInt64, b : Int) -> UInt64 {
+  a.to_int64().lsl(b).to_uint64()
+}
```

Comment:
@Yoorkin would UInt64 provide its own lsl?

Reply:
Yes, the bitwise operations for unsigned integers will be available on today's release.